src/config.rs

```diff
@@ -41,6 +41,33 @@ impl Device {
         }
         Ok(())
     }
+
+    fn is_enabled(&self, memtotal_mb: u64) -> bool {
+        if self.host_memory_limit_mb.is_some() && self.host_memory_limit_mb.unwrap() < memtotal_mb {
```

Comment:
This can be written as:
```rust
let enabled = match limit {
    Some(l) if l < total => { println!(...); false },
    _ => true,
};
return enabled;
```

However I'm unsure about the some-and-not-lt case. Does a zero limit mean enabled here?

Reply:
> Does a zero limit mean enabled here?

0 means disabled, None means enabled.